index.js

```diff
@@ -110,7 +110,19 @@ function dragDrop (elem, listeners) {
     }
 
     // file drop support
-    if (e.dataTransfer.items) {
+    if (e.dataTransfer.files) {
+      var files = e.dataTransfer.files
```

Comment:
Changing the return type here is a breaking change.

Reply:
it's also the only reason to make this change, if the return type doesn't change, this library can't be used to transfer a file drag and drop to a file input